pkg/agent/flowexporter/connections/conntrack_linux.go

```diff
@@ -0,0 +1,142 @@
+// +build linux
+
+package connections
+
+import (
+	"k8s.io/klog"
+
+	"github.com/ti-mo/conntrack"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/config"
+	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter"
+	"github.com/vmware-tanzu/antrea/pkg/agent/openflow"
+)
+
+var _ ConnTrackPoller = new(connTrackPoller)
+
+type connTrackPoller struct {
+	nodeConfig *config.NodeConfig
+	conntrack  ConnTrack
+}
+
+func NewConnTrackPoller(nodeConfig *config.NodeConfig, conntrack ConnTrack) *connTrackPoller {
+	return &connTrackPoller{
+		nodeConfig,
+		conntrack,
+	}
+}
+
+func (cp *connTrackPoller) DumpFlows(zoneFilter uint16) ([]*flowexporter.Connection, error) {
```

Comment:
add function comment with some high-level information about the implementation (e.g. DumpFlows uses netlink to ...)